jerry-core/parser/js/js-parser.c

```diff
@@ -20,6 +20,8 @@
 #include "jcontext.h"
 #include "js-parser-internal.h"
 
+#ifndef CONFIG_DISABLE_PARSER_BUILTIN
```

Comment:
Parser is not exactly a built-in. JERRY_DISABLE_PARSER would sound better. Shouldn't we guard the other files too?